docs/core/tools/dotnet-dev-certs.md

```diff
@@ -0,0 +1,186 @@
+---
+title: dotnet dev-certs command
+description: The dotnet dev-certs command generates a self-signed certificate to enable HTTPS use in development.
+ms.date: 06/06/2022
+---
+# dotnet dev-certs
+
+**This article applies to:** ✔️ .NET Core 3.1 SDK and later versions
+
+## Name
+
+`dotnet dev-certs` - Generates a self-signed certificate to enable HTTPS use in development.
+
+## Synopsis
+
+```dotnetcli
+dotnet dev-certs https 
+  [-c|--check] [--clean] [-ep|--export-path]
+  [--format] [-i|--import] [-np|--no-password]
+  [-p|--password] [-q|--quiet] [-t|--trust]
+  [-v|--verbose] [--version]
+
+dotnet dev-certs https -h|--help
+```
+
+## Description
+
+The `dotnet dev-certs` command generates a self-signed certificate to enable HTTPS use in local web app development. The command recognizes options that perform related functions, such as checking if a certificate is already present, and importing and exporting certificates from and to certificate files.
+
+## Commands
+
+<!-- markdownlint-disable MD012 -->
+
+- **`https`**
+
+  The `dotnet dev-certs` command has only one subcommand: `https`. The `dotnet dev-certs https` command with no options checks if a development certificate is present in the user's certificate store on the machine. If the command finds a development certificate, it displays a message like the following example:
+
+  ```output
+  A valid HTTPS certificate is already present.
+  ```
+
+  If the command doesn't find a development certificate:
+
+  * On Windows, it creates a certificate in the certificate store named `My`, in the location `CurrentUser`. By default, it doesn't create a file. To create a file, use the `--export-path` option.
+  * On Linux, it creates a certificate file and stores the certificate in *~/.dotnet/corefx/cryptography/x509stores/my/*.
+  
+  After creating a certificate, the command displays a message like the following example:
+
+  ```output
+  The HTTPS developer certificate was generated successfully.
+  ```
+
+  By default, the newly created certificate is not trusted. To trust the certificate, use the `--trust` option.
+
+## Options
+
+- **`-c|--check`**
+
+  Checks for the existence of the development certificate but doesn't perform any action. Use this option with the `--trust` option to check if the certificate is not only valid but also trusted.
+
+- **`--clean`**
+
+  Removes all HTTPS development certificates from the machine. On Windows, it doesn't get rid of any physical files that have been exported. It only clears the certificate store of the generated certificate.
```

Comment:
Removes all HTTPS development certificates from the certificate store.

> On Windows, it doesn't get rid of any physical files that have been exported

On Windows, it doesn't get rid of any physical files that have been exported. It only clears the certificate store of the generated certificate.

This is confusing, I would omit any reference to exported files because these two behaviors are orthogonal.

Reply:
Exporting the cert is meant to allow it to be consumed by other tools.